src/jobtypes.py

```diff
@@ -68,11 +67,13 @@ def __init__(self, repo_name='mozilla-inbound'):
                 #  opt test web-platform-tests-4"
                 if job['build_system_type'] == 'buildbot':
                     testtype = job['ref_data_name'].split(' ')[-1]
+                    buildername = job['ref_data_name']
 
                 # taskcluster's testtype is a part of its 'job_type_name' like reftest-2
                 # for [TC] Linux64 reftest-2
                 else:
                     testtype = job['job_type_name'].split(' ')[-1]
+                    buildername = job['job_type_name']
```

Comment:
IMHO it is better to name variables that are agnostic to both systems.
In Buildbot we have buildernames while on TaskCluster we have tasks labels.
A variable that would be indepdent to both systems would be job_identifier or job_uuid.

Note that we also have task label (which are unique) instead of task names (which is defined on the tree but the developer can by mistake name two tasks with the same name).